index.html

```diff
@@ -7118,6 +7119,12 @@ <h2>Changes since Recommendation of 16 July 2020</h2>
       14 to begin with <q>Otherwise</q>.</li>
     <li>Use <a>code point order</a> as a better definition for the more vague "lexicographical order".</li>
     <li>Remove spurious paragraph in the description of {{JsonLdOptions/frameExpansion}}</li>
+    <li>Specify that the default value of <a data-link-for="LoadDocumentOptions">extractAllScripts</a>
```

Comment:
Redundant with "defaults to `true`..." just below
```suggestion
    <li>Specify that the value of <a data-link-for="LoadDocumentOptions">extractAllScripts</a>
```